src/vision/status_quo/frederik_makes_their_first_steps_into_async.md

```diff
@@ -0,0 +1,64 @@
+
+### The wrong time for big decisions
+Frederik's dreams to quickly get their hands dirty with async Rust are shattered as they discover that they first need to make a big choice: what executor to use. Having had quite a bit of exposure to the conversations surrounding the incompatible ecosystems, Frederik is perhaps a bit more paranoid about making the wrong choice than the average newcomer.
+This feels like a big decision to them, as it would influence the libraries they could use and switching to a different ecosystem would be all but impossible after a while. Since they would like to choose what libraries they use before having to choose an executor, Frederik feels like the decision making is turned on its head. 
+
+Their paranoia about choosing the right ecosystem is eased after a few days of research, and some more conversations on the Rust subreddit, after which they discover that most of the RPC libraries they might want to use are situated within the most popular Tokio ecosystem anyways. Tokio also has a brief tutorial, which teaches them some basic concepts within Tokio and talks a bit more about async in general.
+
+### Woes of a newcomer to async
+Being reasonably confident in their choice of ecosystem, Frederik starts building their distributed system.
+After a while, they want to introduce another networking library of which the api isn't async. Luckily Frederik picked up on that blocking was not allowed in async (or at least not in any of the currently existing executors), through reading some blog posts about async. More reddit discussions point them towards spawn\_blocking in Tokio, and even rayon. But they're none the wiser about how to apply these paradigms in a neat manner.
+
+Previously the design patterns learned in other languages, combined with the patterns thaught in the book, were usually sufficient to come to reasonably neat designs.
+But neither their previous experience, nor the async book nor the Tokio tutorial were of much use when trying to neatly incorporate blocking code into their previously fully async project.
+
+### Confused ever after
+To this day the lack of a blessed approach leaves Frederik unsure about the choices they've made so far and misconceptions they might still have, evermore wondering if the original tweets they read about most how most people should just stick to threads were right all along.
+
+🤔 Frequently Asked Questions
+
+* **What are the morals of the story?**
+    * When entering Rust's async world without previous async experience, and no benchmarks for what good async design patters look like, getting started with async can be a bit overwhelming.
+    * Other languages which only have a single ecosystem seem to have a much better story for beginners since there's no fear of lock in, or ecosystem fomo about making the wrong choices early on.
+* **What are the sources for their story?**
```

Comment:
While this list is certainly interesting, I think the question is meant to answer what is the source of the story itself which from your PR sounds like is just personal experience.